[2025-04-11T04:13:52.240+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-11T04:13:52.256+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.data_loading scheduled__2025-04-10T00:00:00+00:00 [queued]>
[2025-04-11T04:13:52.263+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.data_loading scheduled__2025-04-10T00:00:00+00:00 [queued]>
[2025-04-11T04:13:52.265+0000] {taskinstance.py:2856} INFO - Starting attempt 19 of 20
[2025-04-11T04:13:52.276+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): data_loading> on 2025-04-10 00:00:00+00:00
[2025-04-11T04:13:52.285+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=134) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-11T04:13:52.288+0000] {standard_task_runner.py:72} INFO - Started process 140 to run task
[2025-04-11T04:13:52.286+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'data_loading', 'scheduled__2025-04-10T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/dag_for_workshop.py', '--cfg-path', '/tmp/tmp6x91fbzu']
[2025-04-11T04:13:52.289+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask data_loading
[2025-04-11T04:13:52.331+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.data_loading scheduled__2025-04-10T00:00:00+00:00 [running]> on host d3a7fa6095e6
[2025-04-11T04:13:53.638+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='example@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='data_loading' AIRFLOW_CTX_EXECUTION_DATE='2025-04-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='19' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-10T00:00:00+00:00'
[2025-04-11T04:13:53.639+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-11T04:13:54.642+0000] {load.py:25} INFO - Loading clean data to the database.
[2025-04-11T04:13:54.643+0000] {db_operations.py:21} INFO - Connecting to database database workshop_2 , user workshop2, host dpg-cvrve4q4d50c73d7h19g-a.ohio-postgres.render.com, port 5432, password O0mkYGKfG2jhnjOCxKKnhDE2dhLgxaEg
[2025-04-11T04:13:57.232+0000] {db_operations.py:51} INFO - Attempting to create/load table merged_data from Pandas DataFrame
[2025-04-11T04:13:59.179+0000] {db_operations.py:55} INFO - Table merged_data does not exist. Creating it...
[2025-04-11T04:14:00.062+0000] {db_operations.py:69} INFO - Table schema for merged_data created.
